app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::API
+  include ActionController::Serialization
+
+  def start_id
+    params[:start_id].to_i
+  end
+
+  def count
+    params[:count] || default_count
+  end
```

Comment:
why are these methods on ApplicationController? seems like they're only relevant for the events index